activerecord/test/cases/migration/columns_test.rb

```diff
@@ -321,6 +321,26 @@ def test_add_column_without_column_name
       ensure
         connection.drop_table :my_table, if_exists: true
       end
+
+      def test_remove_columns_single_statement
+        connection.create_table "my_table" do |t|
+          t.integer "col_one"
+          t.integer "col_two"
+        end
+
+        queries = capture_sql { connection.remove_columns("my_table", "col_one", "col_two") }
+
+        columns = connection.columns("my_table").map(&:name)
+        assert_equal ["id"], columns
+
+        if current_adapter?(:Mysql2Adapter, :PostgreSQLAdapter)
+          assert_equal 1, queries.length
```

Comment:
There's an `assert_queries` that you should be able to use I think.

Reply:
Fixed in "Use assert_queries"